src/wasm-interpreter.h

```diff
@@ -491,6 +491,22 @@ class ExpressionRunner : public Visitor<SubType, Flow> {
   }
 };
 
+// Execute an expression in global init
+class GlobalInitRunner : public ExpressionRunner<GlobalInitRunner> {
+public:
+  Flow visitLoop(Loop* curr) { WASM_UNREACHABLE(); }
```

Comment:
This should probably result in some kind of validation error, rather than WASM_UNREACHABLE (since otherwise well-formed input could reach this code?)